packages/bulkapi/src/database/QueryNode.ts

```diff
@@ -98,7 +98,7 @@ export const nodesForQueryQuery = (nodesQuery: string): string => {
             )
         containments
         FROM node_properties n1
-        LEFT JOIN ${CONTAINMENTS_TABLE} con  ON con.node_id  = n1.id 
+        LEFT JOIN ${CONTAINMENTS_TABLE} con  ON con.node_id  = n1.id  
```

Comment:
extra white space at the end of the line

Reply:
Removed